src/other/precompile.jl

```diff
@@ -0,0 +1,3216 @@
+# precompile.jl contains precompilation directives for all methods whose compilation
+# is triggered by running DataFrames tests and takes more than a given threshold
+# For all=true, the threshold is 0.01, for all=false it is 0.2.
+# `precompile` calls are generated using SnoopCompile via:
+# using SnoopCompileCore
+# inf_timing = @snoopi tmin=0.01 include("test/runtests.jl")
+# using SnoopCompile
+# pc = SnoopCompile.parcel(inf_timing)
+# SnoopCompile.write("src/other/precompile_tmp.jl", pc[:DataFrames], always=true)
```

Comment:
maybe pls. add comments how you manually tweaked it so that later it is easier to repeat the process? Thank you!